src/features/common/RedeemMicro/RedeemCode.tsx

```diff
@@ -64,23 +64,21 @@ export const InputRedeemCode = ({
           placeholder="XAD-1SA-5F1-A"
           label=""
           variant="outlined"
+          error={errors.code}
+          helperText={errors.code && errors.code.message}
         />
```

Comment:
Should we move to the field styles similar to Edit Profile? i.e. use TextField instead of MaterialTextField? We're moving away from MaterialTextField